pkg/appgw/backendhttpsettings.go

```diff
@@ -31,6 +31,11 @@ func (c *appGwConfigBuilder) BackendHTTPSettingsCollection(cbCtx *ConfigBuilderC
 	if httpSettings != nil {
 		sort.Sort(sorter.BySettingsName(*httpSettings))
 	}
+
+	if cbCtx.EnvVariables.EnableBrownfieldDeployment == "true" {
+		// TODO(draychev): implement
```

Comment:
This (and the other TODOs) - deliberately left here to show what the missing pieces of the `brownfield deployment` work are.